File: Cryptofolio/ClientApp/src/app/common/models/voting-history.ts
export class VotingHistory{
    public Id : number = -1;

    public Status: VoteStatus = VoteStatus.Unknown;

    public Date : Date = new Date;

    public  CoinSymbol : string = "";

    public  ApplicationUserId : string = "";

    public constructor(votingHistory?: VotingHistory) {
      if (votingHistory != undefined){
      this.Id = votingHistory.Id;
      this.Status = votingHistory.Status;
      this.Date = votingHistory.Date;
      this.CoinSymbol = votingHistory.CoinSymbol;
      this.ApplicationUserId = votingHistory.ApplicationUserId;
      }
  }
}
 export enum VoteStatus{
    Unknown = "Unknown",
    Bullish = "Bullish",
    Bearish = "Bearish"
}

File: Cryptofolio/Models/VotingHistory.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cryptofolio.Models
{
    public class VotingHistory
    {
        [Key]
        public int Id { get; set; }

        public VoteStatus Status { get; set; } = VoteStatus.Unknown;

        public DateTime Date { get; set; }
        public string CoinSymbol { get; set; } = string.Empty;

        [ForeignKey("CoinSymbol")]
        public Coin? Coin { get; set; }

        public string? ApplicationUserId { get; set; }

        [ForeignKey("ApplicationUserId")]
        public ApplicationUser? ApplicationUser { get; set; }
    }

    public enum VoteStatus
    {
        Unknown,
        Bullish,
        Bearish
    }
}
